### PR TITLE
Corrected spelling and fixed grammar

### PR DESCRIPTION
**Pull request by CaddyDz**
Have nothing else interesting to share xD sorry

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -15,11 +15,11 @@ A curated list of awesome things and projects built by Algerian developpers.
...
 
 ## Applications
-- [Alfanous](https://github.com/Alfanous-team/alfanous) - Alfanous is an Arabic search engine API provides the simple and advanced search in Quran
+- [Alfanous](https://github.com/Alfanous-team/alfanous) - Alfanous is an Arabic search engine API provides a simple and advanced search in Quran
 - [Cryptili](https://github.com/youben11/cryptili) - Cryptili is a lightweight Android app that allows you to encrypt/decrypt messages and send them through a different app.
 - [Ji3an](https://github.com/Taybou/JiaanApp) - Ji3an is a mobile application for food recipes and cart shopping
-- [LSQLApp](https://github.com/houarizegai/lsqlapp) - Desktop application that helps teachers in practical work (TP) to evaluate students. It also helps students to learn SQL.
-- [OpenClass](https://github.com/youben11/open-class/) - OpenClass is a web app that promote information sharing through organized workshops.
+- [LSQLApp](https://github.com/houarizegai/lsqlapp) - a Desktop application that helps teachers in practical work (TP) to evaluate students. It also helps students to learn SQL.
```
Comment (youknowriad) [inline README.md]: Should we uppercase the "a"?

**Review by youknowriad — COMMENTED** [review]
That's great. Thanks for the fixes, just left a minor comment.

File: README.md (hunk 1 above)
Comment (CaddyDz) [reply README.md]: Yes, but then Desktop has to be lower-case then

**Comment by CaddyDz** [comment]
cc // @youknowriad fixed

**Review by youknowriad — APPROVED** [review]
Awesome LGTM 👍

**Pull request merged**